src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/imports/InputCustomImportOrder_MultiplePatternMatches.java

```diff
@@ -0,0 +1,8 @@
+package com.puppycrawl.tools.checkstyle.imports;
```

Comment:
@ivanov-alex sorry, maybe I'm missing something, but why do we need non-existing import for that test? Can't we create file that uses only existing imports?